src/main/java/org/jabref/gui/importer/ImportCustomEntryTypesDialogViewModel.java

```diff
@@ -0,0 +1,64 @@
+package org.jabref.gui.importer;
+
+import java.util.List;
+
+import javafx.beans.property.ListProperty;
+import javafx.beans.property.SimpleListProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
+import org.jabref.Globals;
+import org.jabref.gui.customentrytypes.CustomEntryTypesManager;
+import org.jabref.model.EntryTypes;
+import org.jabref.model.database.BibDatabaseMode;
+import org.jabref.model.entry.CustomEntryType;
+import org.jabref.model.entry.EntryType;
+
+public class ImportCustomEntryTypesDialogViewModel {
+
+    private final ListProperty<EntryType> newTypesProperty;
```

Comment:
There is no need for an `ListProperty` here, an observable list is sufficient (the list property is handy if you want to replace the complete list and need listeners for this).